README.md

```diff
@@ -6,109 +6,87 @@
 [![CircleCI](https://circleci.com/gh/freechipsproject/chisel3/tree/master.svg?style=shield)](https://circleci.com/gh/freechipsproject/chisel3/tree/master)
 [![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/freechipsproject/chisel3.svg?label=release)](https://github.com/freechipsproject/chisel3/releases/latest)
 
-Chisel is a new hardware construction language to support advanced hardware design and circuit generation.
+Chisel is a powerful hardware construction language to support advanced hardware design and circuit generation.
 The latest iteration of [Chisel](https://chisel.eecs.berkeley.edu/) is Chisel3,
-which uses Firrtl as an intermediate hardware representation language.
+which uses Firrtl as an intermediate hardware representation language. Chisel promotes re-use through parameterization and 
```

Comment:
For users, FIRRTL is going to be an implementation detail, though I see value in keeping the link to promote FIRRTL. But do so in a way where it doesn't make people more confused - it shouldn't give the impression that one needs to learn another language to use Chisel.

Verilog should also be mentioned somewhere, because I could see one question being, "all my tools are in Verilog, can I use this for actually (building chips | pushing stuff to FPGA) or is this just a toy design language"

Reply:
Not sure how exactly to do this - making it clear that Chisel can generate Verilog but it's not just a Verilog templating engine is useful (e.g. maybe emphasing the use of OOP, FIRRTL, etc)